internal/provider/cached_image_resource.go

```diff
@@ -301,12 +305,22 @@ func (r *CachedImageResource) Read(ctx context.Context, req resource.ReadRequest
 	img, err := getRemoteImage(data.Image.ValueString())
 	if err != nil {
 		if !strings.Contains(err.Error(), "MANIFEST_UNKNOWN") {
-			resp.Diagnostics.AddError("Error checking remote image", err.Error())
+			// Explicitly not making this an error diag.
```

Comment:
Why?

Reply:
The check for the image not being present is a bit fragile and I'd rather not have a false negative get in the way here.